content/Watsonx/WatsonxGov/Level-4/203.md

```diff
@@ -0,0 +1,155 @@
+    2. From the home screen, click the **hamburger menu** in the top left, then under **AI governance** click **External models**. <br />![external models](../images/203/watsonxgov-l4_external-models.webp)
+    3. On the **External models** page, click the blue **Setup external model governance** button. <br />![setup external gov](../images/203/watsonxgov-l4_setup-external-gov.webp)
+    4. Click the toggle button for **External model governance** to activate it. <br />![external model gov](../images/203/watsonxgov-l4_external-model-gov.webp)
+    5. Click on the **Inventory** dropdown and select the **Platform assets catalog** option.
+    6. Click on the **Default owner** dropdown and select the **admin** user, then click **Apply**. You will return to the previous screen.
+    7. Click the toggle button for **Governance console (IBM OpenPages) integration** to activate it. <br />![openpages switch](../images/203/watsonxgov-l4_openpages-switch.webp)
+    8. On the **Sync with Governance console (IBM OpenPages)** page, click on the **OpenPages location** dropdown and select **IBM Cloud Pak for Data**. Note that, depending on the version you have, this option may be called watsonx instead. <br />![openpages integration](../images/203/watsonxgov-l4_openpages-integration.webp)
+    9.  Click on the **OpenPages users** dropdown and select **admin**, then click **Complete setup**. <br /><br />Integration between watsonx (or Cloud Pak for Data) and OpenPages has now been enabled. Next, you will create an inventory and give the created user access to it. Inventories are collections of model use cases. They are meant to be collaborative so that multiple people that perform different roles can contribute to governance of key assets.<br /><br />
+    10. Click the **Inventories** menu item on the left, and then click the **New inventory** button. <br />![new inventory](../images/203/watsonxgov-l4_new-inventory.webp)
+    11. In the name field, enter `High Oaks Banks Model Use Cases`. Ensure that the **Add collaborators after creation** box is checked, and then click **Create**.
+    12. On the **Set collaborators** page, click **Add collaborators**, then **Add user**, and start typing the user ID of the user you created to find it. Make sure **Admin** is selected, then click add. <br />![add collaborator](../images/203/watsonxgov-l4_add-collaborator.webp)<br />![collaborators](../images/203/watsonxgov-l4_collaborators.webp)<br />Now that you have configured Cloud Pak for Data to use the watsonx governance console (OpenPages), model use cases created in the governance console will appear in the inventory in Cloud Pak for Data, and vice versa. You can close the **Set collaborators** and **Manage** windows to return to the **AI use case** screen, then return to the Cloud Pak for Data home page by clicking the title in the upper left corner.<br /><br />
```

Comment:
These screenshots should be updated to show the `complianceofficer` username to be consistent

`start typing the user ID of the user you created` can specify `complianceofficer`